Degradable Darts, Citadel Style

ZeroNetAlpha opened the pull request:
Just incase upstream doesn't want my memes.

[Changelogs]: ALL foam darts, even our magfoam darts, will be deleted with space cleaner

:cl: ZeroNetAlpha
tweak: Makes all darts deletable with nothing more than a little space cleaner.
tweak: Makes in-flight foam darts dissolve when hit with space cleaner, be it foam, smoke, or a janitor being a badass with a spraybottle.
/:cl:

[why]: Holy shit my autoylathe caused a shit ton of lag that wasn't cleanable by any regular means. Janitor's bane.
https://gyazo.com/a98ed04fb60ae7fcd58f4a8d4aa13d71
ALSO IT'S MODULARIZED.

ZeroNetAlpha committed:
Degradable Darts, Citadel Style

Just incase upstream doesn't want my memes.

ZeroNetAlpha committed:
Merge branch 'master' into Biodegradable-Darts

deathride58 reviewed (dismissed):
How is this not a standard feature tbh

Code looks good

ZeroNetAlpha committed:
Woops.

I wasn't paying enough attention, but it added a feature I wasn't intending. This smooths out the feature.
